extensions/spring-data-rest/deployment/pom.xml

```diff
@@ -41,6 +41,11 @@
             <artifactId>quarkus-resteasy-jsonb-deployment</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-links-deployment</artifactId>
+            <scope>test</scope>
```

Comment:
I had to explicitly add this dependency, I guess because we had to use the `-deployment` version. However, this is not necessary to be added when using `quarkus-spring-data-rest` (there are tests we verify this).